src/routes/follow.ts

```diff
@@ -20,7 +24,12 @@ followRouter.post(
       followingId: Number(currentUserId),
       followedId: Number(userId),
     });
-    res.redirect(String(redirectUrl));
+
+    const activeTab = req.query.activeTab;
+    if (activeTab) {
+      return res.redirect(redirectUrl + "&activeTab=" + activeTab);
+    }
+    res.redirect(redirectUrl);
```

Comment:
optional: URLの結合をしたいときは、標準パッケージの`url`ライブラリをimportして使うといいかもです！

`import { URL } from "node:url";`

```suggestion
    const activeTab = req.query.activeTab;
    if (activeTab) {
      const redirectUrlObj = new URL(redirectUrl);
      redirectUrlObj.searchParams.append("activeTab", activeTab);
      return res.redirect(redirectUrlObj.toString());
    }
    res.redirect(redirectUrl);
```